lib/types/local-date.js

```diff
@@ -172,20 +210,12 @@ class LocalDate {
      * if the given one is greater.
```

Comment:
Uppercase

Reply:
Why uppercase in the middle of a sentence? 

> 0 if they are the same, 1 if the this is greater, and -1 **if** the given one is greater.